homeassistant/components/climate/__init__.py

```diff
@@ -379,8 +379,8 @@ def state_attributes(self):
         data = {
             ATTR_CURRENT_TEMPERATURE:
             self._convert_for_display(self.current_temperature),
-            ATTR_MIN_TEMP: self._convert_for_display(self.min_temp),
```

Comment:
There may be a better way to fix it, but this change does fix ecobee for me and works for both C/F.  Without this change the Celsius temperature is used as F and converted to C, resulting in the slider showing temperatures in the sub 0c range.  Essentially the temperature is incorrectly being converted from already being in C (as it is done here) to C again, but it is assuming it is in F, when it isnt, it's C.  I think the slider is code might be doing a conversion but I havn't dug into it.